diff --git a/swe/channels/system/branding/BRANDING b/swe/channels/system/branding/BRANDING
index ca5e6a1..fe37520 100644
--- a/swe/channels/system/branding/BRANDING
+++ b/swe/channels/system/branding/BRANDING
@@ -5,7 +5,7 @@ PRODUCT_SHORTNAME=Snapdragon
 PRODUCT_INSTALLER_FULLNAME=Snapdragon Installer
 PRODUCT_INSTALLER_SHORTNAME=Snapdragon Installer
 COPYRIGHT=Copyright (c) 2015 The Linux Foundation. All rights reserved.
-PACKAGE_NAME=org.codeaurora.swe.browser
+PACKAGE_NAME=com.chrome.beta
 APK_LABEL=@string/application_name
 APK_AUTHORITIES=browser
 APP_AUTHORITIES=local.browser
